src/alignment/data.py

```diff
@@ -97,8 +97,9 @@ def apply_chat_template(
 
 def get_datasets(
     data_config: DataArguments | dict,
-    splits: List[str] = ["train", "test"],
+    splits: Optional[List[str]] = None,
     configs: Optional[List[str]] = None,
+    text_column: Optional[str] = None,
```

Comment:
Would it be better to specify `columns_to_keep` here and then specify explicitly what we keep in the sft / dpo / cpt scripts? 

I feel this might feel a bit less magic than what I previously did with the columns to keep and is more extensible for future algorithms like kto / orpo etc

Reply:
Is my updated version what you intended? For cpt we just keep the text column, and for sft/dpo we allow all those others that you had defined earlier? @lewtun